### PR TITLE
Fix logic error when pushing.

### PR DESCRIPTION
**Pull request by jmatt**
```
* When files are verified, and they are not public on a pubic
  server, make them public.
* TLDR; Added not to the related if statement.

modified:   lib/git-lfs-s3/application.rb
modified:   lib/git-lfs-s3/version.rb
```

### CODE REVIEW

File: lib/git-lfs-s3/application.rb [hunk 1]
```diff
@@ -101,7 +101,7 @@ def public_read_grant
         status 404
       end
       if settings.public_server and settings.ceph_s3
-        if object.acl.grants.include?(public_read_grant)
+        if not object.acl.grants.include?(public_read_grant)
```
Comment (jhoblitt) [inline lib/git-lfs-s3/application.rb:104]: This makes sense to me but I'm not clear as it why this was working at all before?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ It works by generating a presigned URL that the Git LFS client can use to upload
 Git LFS S3 is available on RubyGems.
 
 ``` bash
-gem install git-lfs-s3
+gem install lsst-git-lfs-s3
```
Comment (jhoblitt) [inline README.md:12]: I have a mild objection to adding changes that will conflict with upstream but I can live with this.

**Comment by jhoblitt** [comment]
:+1:

File: lib/git-lfs-s3/application.rb (hunk 1 above)
Comment (jmatt) [inline lib/git-lfs-s3/application.rb:104]: I don't think we tested push after this refactor. Other than pushing afwdata - which was a one off. I changed permissions of the keys for afwdata. This is why I had to do that extra step.

File: README.md (hunk 2 above)
Comment (jmatt) [inline README.md:12]: I agree. I'm going to remove all the objectionable bits in a branch and push that back upstream.

**Pull request merged**